### ISSUE TITLE
Run dashboard on multiple ports

### ISSUE THREAD

**Issue by jacobtomlinson** [dask/distributed#4782]
In some constrained environments (like one I am currently working with) it may be desirable to run the scheduler dashboard on multiple ports.

The use case I have is that ports on the cluster can only be exposed internally or externally, and it is desirable for the user to connect to the dashboard either way. So the only option is to run the dashboard on two ports and expose one internally and one externally.

My initial though for an implementation would be to allow `dashboard_address` to be a list of comma separated string and to update `start_http_server` and `clean_dashboard_address` to support that.

https://github.com/dask/distributed/blob/233ec8840671b5d705a06d0dcfe1ff42b771230b/distributed/node.py#L99

https://github.com/dask/distributed/blob/233ec8840671b5d705a06d0dcfe1ff42b771230b/distributed/utils.py#L1507

**Comment by mrocklin** [comment]
That seems fine to me.  I think that we should also support a list of
ints.  There is precedent here with other keywords like protocol to have
Dask comms listen on multiple protocols.

I hope that we can just call some listen method a few times on the same
HTTP Server

**Comment by jacobtomlinson** [comment]
> I hope that we can just call some listen method a few times on the same
HTTP Server

That's the plan. 

It just seems like the `clean_dashboard_address` takes a variety of input forms, and we will need to continue supporting those and also support a list of ints, a list of strings like `['localhost:8787', 'localhost:8887']` and also something that can be passed via the CLI like `--dashboard-address ":8787,:8887"`.

**Comment by mrocklin** [comment]
There is always dask-spec.  FWIW we've switched to using this in Coiled and
it's been great.  I think that other dask-foo deployment projects may want
to move over at some point as well.

**Comment by jacobtomlinson** [comment]
Yeah we've used it in Dask Cloudprovider and I'm planning on swapping other things over in the future. But we should probably still support the CLI.
